<launch>
    <!-- 发布原始点云2 -->
    <node pkg="pcl_ros" type="pcd_to_pointcloud" name="cloud2_publisher" output="screen">
        <param name="frame_id" value="map"/>
        <param name="file_name" value="/home/dearmoon/datasets/one_msg/lidar_step2.pcd"/>
        <remap from="output" to="/cloud2"/>
    </node>

    <!-- 发布点云1 -->
    <node pkg="pcl_ros" type="pcd_to_pointcloud" name="cloud1_publisher" output="screen">
        <param name="frame_id" value="map"/>
        <param name="file_name" value="/home/dearmoon/datasets/one_msg/radar_step2.pcd"/>
        <remap from="output" to="/cloud1"/>
    </node>

    <!-- 运行转换节点 -->
    <node pkg="rosbag_tools" type="calibration_test" name="calibration_test" output="screen"/>

    <!-- RViz 可视化 -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find rosbag_tools)/rviz_config/calibration_test.rviz"/>
</launch>
